kernel/relevanceops.ml

```diff
@@ -102,6 +102,7 @@ and relevance_of_term_extra env extra lft subs c =
   | CoFix (i,(lna,_,_)) -> (lna.(i)).binder_relevance
   | Proj (p, _) -> relevance_of_projection env p
   | Int _ | Float _ -> Sorts.Relevant
+  | Array _ -> Sorts.Relevant (* FIXME *)
```

Comment:
@SkySkimmer should something be fixed here?

Reply:
```suggestion
  | Array _ -> Sorts.Relevant
```